### ISSUE TITLE
Styles: Update Chip

### ISSUE THREAD

**Issue by imagoiq** [swisspost/design-system#1864]
## Architecture

```[tasklist]
- [ ] Missing a notion of set or radio group (aka "Filter")
- [ ] Disabled state should be implemented
- [ ] Add radio button variant to the checkbox chip
- [ ] Change naming from badge to chip (deprecate class badge for chips and keep badge for badges)
```
## Filter type styles to change
```[tasklist]
### General
- [ ] Gap between items should be `size-mini` (8px)
- [ ] Check if line-height from design is appropriate 150%. Or use whatever is best for the dev.
```

### Per variant

```[tasklist]
#### Big/default
- [ ] Font should use `font-size-regular` (16px) instead of 15px
- [ ] Padding vertical should be `size-mini` (8px) instead of 0. Or use whatever is best for the dev and relates to line-height.
```

```[tasklist]
#### Small
- [ ] Font should use `font-size-14` instead of 13px
- [ ] Padding horizontal should be `size-small-large` (12px) instead of 7px
```

```[tasklist]
#### Hover
- [ ] Background should be `black-20`
- [ ] Border should `black-60` and size 2px
```

```[tasklist]
#### Disabled and not selected
- [ ] Border should be `black-20` and 2px size
- [ ] Background should be `white`
- [ ] Font-color should be `black-40`
```

```[tasklist]
#### Disabled and selected
- [ ] Font color should be `black-40`
- [ ] Border should be `black-20` and 2px size
- [ ] Background should be `background-dark`
```

```[tasklist]
#### Focused
- [ ] Font color should be `black`
- [ ] Background should be `black-20`
```

#### Counter/nested badge

```[tasklist]
#### Every variant
- [ ] Counter font color should be `badge`
```

```[tasklist]
#### Big
- [ ] Counter has an unneeded negative margin
- [ ] Counter font size should be `font-size-12` not 10px
- [ ] Counter Height and width should be 24px
- [ ] Counter should not have vertical padding. Or use whatever is best for the dev.
```

```[tasklist]
#### Small
- [ ] Gap between icon and text should be 6px not 12px. No need to use spacing variables, use custom value.
- [ ] Counter has an unneeded negative margin
- [ ] Counter Height and Width should be 16px instead of 22px
- [ ] Counter should not have vertical padding. Or use whatever is best for the dev.
```

```[tasklist]
#### Hover
- [ ] Border should be border `black-60` and 2px size
- [ ] Counter should not have a border
- [ ] Counter should have a `white` background
```

```[tasklist]
#### Disabled and not selected
- [ ] Counter background should be `background-dark`
```

```[tasklist]
#### Disabled and selected
- [ ] Counter background should be `white`
```

## Input type styles to change

```[tasklist]
### Disabled and selected
- [x] Dismissible action should be before the text
- [ ] Click area size ? Need to be discussed!
- [ ] Gap between items should be `size-mini` (8px)
- [ ] Icon should come from post (currently from boostrap)
- [ ] Icon should have `black-80` color
- [ ] Padding-start of the icon should be `size-mini` (8px)
- [ ] Icon should not have negative margin
```

### Per variant

```[tasklist]
#### Small
- [ ] Icon size should be `size-large` (16px)
```

```[tasklist]
#### Hover + Focused
- [ ] Round background with `black-20`
```

**Comment by b1aserlu** [comment]
@wueestd @gfellerph @imagoiq 
I noticed that in figma the component chip isnt ready for developement yet. And there have indeed been changes made to the figma file since I last had checked.

Am I right to assume that the task cannot be properly started before the chip component in figma is ready for developement?
Is the task list in this ticket still up to date?
How will the process look like to go from the current badge component to the new badge and chip components?

@oliverschuerch  mentioned that the new chip component might be a canditate that will be implemented as a webcomponent.

**Comment by wueestd** [comment]
I put the task on ready for dev 👍 the only thing that changed since the review is in my mind some Figma Architecture and the focus state of the dismissable chip 

![Image](https://github.com/swisspost/design-system/assets/20660048/e3bc02d4-9a3c-4574-b173-5bc0dc14bc1f)

**Comment by b1aserlu** [comment]
- Deprecate current Badge
- implement new Badge in a tab (web component)
- create new component Chip (web component & normal HTML/CSS)

**Comment by b1aserlu** [comment]
This Issue will be closed in favour of the new Issue #2195 which is a collection of Issues generated from this Issue.
